### ISSUE TITLE
[Bug]: Feedback loop submission results in an error

### ISSUE THREAD

**Issue by heyitsaamir** [microsoft/teams-ai#1956]
### Language

Python

### Version

latest

### Description

Feedback loop values come as camelCased, not snake_cased. Which results in this:
```
 [on_turn_error] unhandled error: 'dict' object has no attribute 'action_name'
Traceback (most recent call last):
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/botbuilder/core/bot_adapter.py", line 174, in run_pipeline
    return await self._middleware.receive_activity_with_status(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/botbuilder/core/middleware_set.py", line 69, in receive_activity_with_status
    return await self.receive_activity_internal(context, callback)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/botbuilder/core/middleware_set.py", line 79, in receive_activity_internal
    return await callback(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 714, in on_turn
    await self._start_long_running_call(context, self._on_turn)
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 871, in _start_long_running_call
    return await func(context)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 732, in _on_turn
    is_ok, matches = await self._on_activity(context, state)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 841, in _on_activity
    for route in invoke_routes:
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 839, in <lambda>
    invoke_routes = filter(lambda r: r.is_invoke and r.selector(context), self._routes)
                                                     ^^^^^^^^^^^^^^^^^^^
  File "/Users/aamirjawaid/miniconda3/envs/privacy-helper/lib/python3.12/site-packages/teams/app.py", line 602, in __selector__
    and context.activity.value.action_name == "feedback"
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'action_name'
```
Here is the actual payload.
![image](https://github.com/user-attachments/assets/889125a5-5626-4e64-aa8e-42b1a35a82cc)

### Reproduction Steps

```bash
1. Send a feedback loop feedback.
2.
3.
...
```

**Comment by lilyydu** [comment]
hey @heyitsaamir, we were trying to fix this via #1943 and #1892 but I wasn't able to test- would you mind creating a PR fix for this and testing on your end?

was under the assumption that it was snake-cased cause the internal botbuilder schema model often converts everything back into snake-cased but strange this isn't the case

**Comment by heyitsaamir** [comment]
> hey @heyitsaamir, we were trying to fix this via #1943 and #1892 but I wasn't able to test- would you mind creating a PR fix for this and testing on your end?
> 
> was under the assumption that it was snake-cased cause the internal botbuilder schema model often converts everything back into snake-cased but strange this isn't the case

Check the attached PR. There's another issue here where the feedback loop continuously retries. I don't understand why that's happening
